pentasharp/Services/AdminSetupService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Security.Cryptography;
+using System.Text;
+using pentasharp.Data;
+using pentasharp.Models.Entities;
+
+namespace pentasharp.Services
+{
+    public class AdminSetupService
+    {
+        private readonly AppDbContext _context;
+
+        public AdminSetupService(AppDbContext context)
+        {
+            _context = context;
+        }
+
+        public void EnsureAdminUserExists()
+        {
+            if (!_context.Users.Any(u => u.IsAdmin))
+            {
+                var adminUser = new User
+                {
+                    FirstName = "Penta",
```

Comment:
Use AutoMapper for object mappings instead of manually assigning properties.

Reply:
you introduced mapper here, but you are still not using it, 
if you need reference similar change was done by @oltahasimja in this file: pentasharp/Controllers/AuthenticateController.cs